packages/infolists/resources/views/components/text-entry.blade.php

```diff
@@ -37,6 +37,10 @@
         }
 
         $arrayState = \Illuminate\Support\Arr::wrap($arrayState);
+
+        if (blank($arrayState) && filled($getPlaceholder())) {
+            $arrayState = \Illuminate\Support\Arr::prepend($arrayState, '');
```

Comment:
Doesn't this need to prepend the actual placeholder as state instead of an empty string? Also, can't it just reassign `$arrayState` completely into a new array since its already blank, instead of prepending

Reply:
Lets use `null` instead of an empty string please

Reply:
Potentially, we don't want the placeholder to run through `formatState()` though 🤔 It might need different styling, for example. Maybe we should use an if statement in the view to decide whether or not to render a placeholder. And then remove the placeholder application from the `formatState()` method

Reply:
I think that looking at how the placeholder is formatted might be for another PR?

This PR was about trying to address how a placeholder works as intended only if the state is an empty string but not if the state is `null`.